WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
ERROR demo.py:14 error info
WARNING demo.py:15 warn info
INFO demo.py:16 info info
DEBUG demo.py:17 debug info
WARNING _internal.py:88  * Debugger is active!
ERROR demo.py:14 error info
WARNING demo.py:15 warn info
INFO demo.py:16 info info
DEBUG demo.py:17 debug info
WARNING _internal.py:88  * Debugger is active!
ERROR demo.py:14 error info
WARNING demo.py:15 warn info
INFO demo.py:16 info info
DEBUG demo.py:17 debug info
ERROR demo.py:14 error info
WARNING demo.py:15 warn info
INFO demo.py:16 info info
DEBUG demo.py:17 debug info
WARNING _internal.py:88  * Debugger is active!
ERROR demo.py:15 error info
WARNING demo.py:16 warn info
INFO demo.py:17 info info
DEBUG demo.py:18 debug info
ERROR demo.py:15 error info
WARNING demo.py:16 warn info
INFO demo.py:17 info info
DEBUG demo.py:18 debug info
ERROR demo.py:15 error info
WARNING demo.py:16 warn info
INFO demo.py:17 info info
DEBUG demo.py:18 debug info
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
ERROR demo.py:15 error info
WARNING demo.py:16 warn info
INFO demo.py:17 info info
DEBUG demo.py:18 debug info
ERROR __init__.py:97 Error: Path doesn't exist: 'migrations'.  Please use the 'init' command to create a new scripts folder.
ERROR __init__.py:97 Error: Path doesn't exist: 'migrations'.  Please use the 'init' command to create a new scripts folder.
ERROR __init__.py:97 Error: Path doesn't exist: 'migrations'.  Please use the 'init' command to create a new scripts folder.
WARNING _internal.py:88  * Debugger is active!
ERROR demo.py:14 error info
WARNING demo.py:15 warn info
INFO demo.py:16 info info
DEBUG demo.py:17 debug info
ERROR demo.py:14 error info
WARNING demo.py:15 warn info
INFO demo.py:16 info info
DEBUG demo.py:17 debug info
ERROR __init__.py:97 Error: Path doesn't exist: 'migrations'.  Please use the 'init' command to create a new scripts folder.
ERROR demo.py:14 error info
WARNING demo.py:15 warn info
ERROR demo.py:14 error info
WARNING demo.py:15 warn info
WARNING _internal.py:88  * Debugger is active!
ERROR demo.py:14 error info
WARNING demo.py:15 warn info
INFO demo.py:16 info info
DEBUG demo.py:17 debug info
ERROR __init__.py:97 Error: Path doesn't exist: 'migrations'.  Please use the 'init' command to create a new scripts folder.
ERROR __init__.py:97 Error: Path doesn't exist: 'migrations'.  Please use the 'init' command to create a new scripts folder.
ERROR __init__.py:97 Error: Path doesn't exist: 'migrations'.  Please use the 'init' command to create a new scripts folder.
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
ERROR verify_code.py:23 value is not an integer or out of range
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
WARNING _internal.py:88  * Debugger is active!
2018-12-21 17:47:40,795 WARNING _internal.py:88  * Debugger is active!
2018-12-21 17:47:51,610 WARNING _internal.py:88  * Debugger is active!
2018-12-21 17:49:42,845 WARNING _internal.py:88  * Debugger is active!
2018-12-21 18:07:09,703 WARNING _internal.py:88  * Debugger is active!
2018-12-21 18:07:50,455 WARNING _internal.py:88  * Debugger is active!
2018-12-21 18:10:29,016 WARNING _internal.py:88  * Debugger is active!
2018-12-21 18:10:39,623 ERROR verify_code.py:55 'bytes' object has no attribute 'encode'
2018-12-21 18:11:24,768 WARNING _internal.py:88  * Debugger is active!
2018-12-21 18:11:29,468 ERROR verify_code.py:89 When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-24 09:06:43,477 WARNING _internal.py:88  * Debugger is active!
2018-12-24 09:09:39,220 ERROR verify_code.py:89 When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-24 09:32:53,689 WARNING _internal.py:88  * Debugger is active!
2018-12-24 09:33:05,264 WARNING _internal.py:88  * Debugger is active!
2018-12-24 10:14:17,189 WARNING _internal.py:88  * Debugger is active!
2018-12-24 10:43:03,546 WARNING _internal.py:88  * Debugger is active!
2018-12-24 10:47:19,222 WARNING _internal.py:88  * Debugger is active!
2018-12-24 10:49:35,679 ERROR verify_code.py:56 'NoneType' object has no attribute 'decode'
2018-12-24 10:51:15,190 WARNING _internal.py:88  * Debugger is active!
2018-12-24 10:51:25,542 WARNING _internal.py:88  * Debugger is active!
2018-12-24 10:55:14,205 WARNING _internal.py:88  * Debugger is active!
2018-12-24 11:02:16,241 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:13:03,603 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:14:43,186 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:14:57,421 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:15:25,880 ERROR verify_code.py:90 When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-25 10:17:33,523 ERROR verify_code.py:90 One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'Mapper|User|ih_user_profile'. Original exception was: When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-25 10:19:32,563 ERROR verify_code.py:90 One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'Mapper|User|ih_user_profile'. Original exception was: When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-25 10:22:26,527 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:22:46,331 ERROR verify_code.py:90 When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-25 10:25:22,186 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:25:35,540 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:25:44,623 ERROR verify_code.py:90 When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-25 10:26:45,091 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:26:57,679 ERROR verify_code.py:90 When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-25 10:28:02,534 ERROR verify_code.py:90 One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'Mapper|User|ih_user_profile'. Original exception was: When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-25 10:28:56,084 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:28:59,117 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:29:18,865 ERROR verify_code.py:90 When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-25 10:31:32,303 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:31:37,970 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:32:03,002 ERROR verify_code.py:90 When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-25 10:40:49,828 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:40:53,573 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:41:10,389 ERROR verify_code.py:90 When initializing mapper Mapper|User|ih_user_profile, expression 'House' failed to locate a name ("name 'House' is not defined"). If this is a class name, consider adding this relationship() to the <class 'ihome.models.User'> class after both dependent classes have been defined.
2018-12-25 10:52:57,152 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:53:51,789 WARNING _internal.py:88  * Debugger is active!
2018-12-25 10:56:06,574 WARNING _internal.py:88  * Debugger is active!
2018-12-25 11:08:14,483 WARNING _internal.py:88  * Debugger is active!
2018-12-25 11:08:39,885 WARNING _internal.py:88  * Debugger is active!
2018-12-25 11:10:29,754 WARNING _internal.py:88  * Debugger is active!
2018-12-25 11:11:07,384 WARNING _internal.py:88  * Debugger is active!
2018-12-25 11:12:58,420 WARNING _internal.py:88  * Debugger is active!
2018-12-25 11:14:43,666 WARNING _internal.py:88  * Debugger is active!
2018-12-26 10:19:58,147 WARNING _internal.py:88  * Debugger is active!
2018-12-26 10:22:47,008 WARNING _internal.py:88  * Debugger is active!
2018-12-26 10:23:05,464 WARNING _internal.py:88  * Debugger is active!
2018-12-26 10:23:15,028 WARNING _internal.py:88  * Debugger is active!
2018-12-26 10:25:46,270 ERROR passport.py:122 (pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'localhost' ([WinError 10061] ����Ŀ�����������ܾ����޷����ӡ�)") (Background on this error at: http://sqlalche.me/e/e3q8)
2018-12-27 08:53:37,022 WARNING _internal.py:88  * Debugger is active!
2018-12-27 10:00:10,840 WARNING _internal.py:88  * Debugger is active!
2018-12-27 10:39:32,765 WARNING _internal.py:88  * Debugger is active!
2018-12-27 10:40:39,583 WARNING _internal.py:88  * Debugger is active!
2018-12-27 10:40:44,941 WARNING _internal.py:88  * Debugger is active!
2018-12-27 11:01:30,705 WARNING _internal.py:88  * Debugger is active!
2018-12-27 11:09:41,437 WARNING _internal.py:88  * Debugger is active!
2018-12-27 11:10:12,699 WARNING _internal.py:88  * Debugger is active!
2018-12-27 11:15:36,724 WARNING _internal.py:88  * Debugger is active!
2018-12-27 12:21:52,307 WARNING _internal.py:88  * Debugger is active!
2018-12-27 12:23:49,279 WARNING _internal.py:88  * Debugger is active!
2018-12-27 12:38:19,033 WARNING _internal.py:88  * Debugger is active!
2018-12-27 12:40:00,615 WARNING _internal.py:88  * Debugger is active!
2018-12-27 12:40:39,420 WARNING _internal.py:88  * Debugger is active!
2018-12-27 12:43:13,196 WARNING _internal.py:88  * Debugger is active!
2018-12-27 12:43:45,825 WARNING _internal.py:88  * Debugger is active!
2018-12-27 12:47:37,029 WARNING _internal.py:88  * Debugger is active!
2018-12-27 12:47:43,074 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:02:15,460 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:06:09,730 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:09:22,597 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:10:55,732 ERROR profile.py:90 Entity '<class 'ihome.models.User'>' has no property 'avatar'
2018-12-27 16:12:43,354 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:12:47,388 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:22:33,254 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:37:50,656 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:41:46,145 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:41:55,343 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:42:07,979 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:42:57,629 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:43:01,057 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:43:47,176 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:44:20,643 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:50:36,577 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:50:46,934 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:51:14,664 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:52:43,263 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:53:01,657 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:53:53,502 WARNING _internal.py:88  * Debugger is active!
2018-12-27 16:53:58,115 WARNING _internal.py:88  * Debugger is active!
2018-12-28 09:24:06,080 WARNING _internal.py:88  * Debugger is active!
2018-12-28 09:30:46,321 WARNING _internal.py:88  * Debugger is active!
2018-12-28 09:59:54,800 WARNING _internal.py:88  * Debugger is active!
2018-12-28 10:10:08,781 WARNING _internal.py:88  * Debugger is active!
2018-12-28 10:11:45,861 WARNING _internal.py:88  * Debugger is active!
2018-12-28 10:13:25,128 WARNING _internal.py:88  * Debugger is active!
2018-12-28 10:19:09,920 WARNING _internal.py:88  * Debugger is active!
2018-12-28 10:19:14,674 WARNING _internal.py:88  * Debugger is active!
2018-12-28 10:19:35,700 WARNING _internal.py:88  * Debugger is active!
2018-12-28 10:20:46,780 WARNING _internal.py:88  * Debugger is active!
2018-12-28 10:20:52,513 WARNING _internal.py:88  * Debugger is active!
2018-12-28 10:25:00,525 WARNING _internal.py:88  * Debugger is active!
2018-12-28 10:44:09,541 WARNING _internal.py:88  * Debugger is active!
2018-12-28 16:39:24,421 WARNING _internal.py:88  * Debugger is active!
2018-12-28 16:54:44,216 WARNING _internal.py:88  * Debugger is active!
2018-12-28 16:55:46,104 WARNING _internal.py:88  * Debugger is active!
2018-12-28 16:55:47,519 INFO houses.py:24 hit redis area_info
2018-12-28 16:55:57,377 INFO houses.py:24 hit redis area_info
2018-12-28 16:56:57,726 WARNING _internal.py:88  * Debugger is active!
2018-12-28 16:56:59,641 INFO houses.py:24 hit redis area_info
2018-12-28 17:06:07,726 WARNING _internal.py:88  * Debugger is active!
2018-12-29 09:13:25,507 WARNING _internal.py:88  * Debugger is active!
2018-12-29 09:33:06,497 WARNING _internal.py:88  * Debugger is active!
2018-12-29 09:33:10,959 INFO houses.py:24 hit redis area_info
2018-12-29 09:34:05,426 INFO houses.py:24 hit redis area_info
2018-12-29 11:04:33,717 WARNING _internal.py:88  * Debugger is active!
2018-12-29 11:04:56,487 INFO houses.py:25 hit redis area_info
2018-12-29 11:10:42,597 INFO houses.py:25 hit redis area_info
2018-12-29 11:14:59,464 WARNING _internal.py:88  * Debugger is active!
2018-12-29 11:19:21,869 WARNING _internal.py:88  * Debugger is active!
2018-12-29 11:19:30,064 WARNING _internal.py:88  * Debugger is active!
2018-12-29 11:19:39,450 WARNING _internal.py:88  * Debugger is active!
2018-12-29 11:19:45,430 WARNING _internal.py:88  * Debugger is active!
2018-12-29 11:19:58,398 WARNING _internal.py:88  * Debugger is active!
2018-12-29 11:23:38,502 WARNING _internal.py:88  * Debugger is active!
2018-12-29 11:24:20,582 WARNING _internal.py:88  * Debugger is active!
2018-12-29 11:26:03,736 WARNING _internal.py:88  * Debugger is active!
2018-12-29 13:54:05,576 WARNING _internal.py:88  * Debugger is active!
2018-12-29 13:56:44,404 WARNING _internal.py:88  * Debugger is active!
2018-12-29 13:59:39,729 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:01:01,018 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:02:09,899 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:05:54,324 INFO houses.py:25 hit redis area_info
2018-12-29 14:10:49,851 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:17:37,970 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:17:43,673 INFO houses.py:25 hit redis area_info
2018-12-29 14:17:57,313 INFO houses.py:25 hit redis area_info
2018-12-29 14:18:44,163 INFO houses.py:25 hit redis area_info
2018-12-29 14:24:28,066 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:24:44,173 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:25:26,785 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:26:22,237 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:26:33,746 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:26:59,154 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:27:04,827 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:27:18,519 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:27:29,967 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:29:09,845 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:30:46,191 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:30:56,439 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:31:36,465 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:31:52,435 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:32:10,715 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:32:28,966 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:32:39,162 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:33:29,436 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:33:52,203 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:52:32,016 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:53:19,069 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:57:53,031 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:58:21,729 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:58:49,203 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:58:56,119 INFO houses.py:24 hit redis area_info
2018-12-29 14:59:13,311 WARNING _internal.py:88  * Debugger is active!
2018-12-29 14:59:14,792 INFO houses.py:24 hit redis area_info
2018-12-29 15:05:25,130 WARNING _internal.py:88  * Debugger is active!
2018-12-29 15:05:25,447 INFO houses.py:24 hit redis area_info
2018-12-29 15:07:56,006 WARNING _internal.py:88  * Debugger is active!
2018-12-29 15:16:24,362 INFO houses.py:24 hit redis area_info
2018-12-29 15:16:28,299 INFO houses.py:24 hit redis area_info
2018-12-29 15:17:46,532 INFO houses.py:24 hit redis area_info
2018-12-29 15:25:13,778 WARNING _internal.py:88  * Debugger is active!
2018-12-29 15:26:03,265 WARNING _internal.py:88  * Debugger is active!
2018-12-29 15:52:31,718 WARNING _internal.py:88  * Debugger is active!
2018-12-29 16:02:04,602 WARNING _internal.py:88  * Debugger is active!
2019-01-02 15:38:24,454 WARNING _internal.py:88  * Debugger is active!
2019-01-02 15:57:06,849 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:04:13,060 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:06:53,354 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:19:04,781 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:23:04,194 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:32:10,227 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:32:13,854 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:32:56,210 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:34:30,720 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:37:08,499 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:41:38,156 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:42:18,679 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:42:24,733 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:43:04,816 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:43:40,595 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:43:47,750 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:51:42,414 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:52:40,944 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:52:58,772 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:53:03,711 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:53:15,600 WARNING _internal.py:88  * Debugger is active!
2019-01-02 16:53:58,582 WARNING _internal.py:88  * Debugger is active!
2019-01-02 17:00:58,400 WARNING _internal.py:88  * Debugger is active!
2019-01-02 17:02:50,634 WARNING _internal.py:88  * Debugger is active!
2019-01-02 17:02:58,967 WARNING _internal.py:88  * Debugger is active!
2019-01-02 17:07:43,211 WARNING _internal.py:88  * Debugger is active!
2019-01-02 17:07:50,374 WARNING _internal.py:88  * Debugger is active!
2019-01-02 17:08:04,544 WARNING _internal.py:88  * Debugger is active!
2019-01-02 17:08:27,086 WARNING _internal.py:88  * Debugger is active!
2019-01-02 17:08:38,895 WARNING _internal.py:88  * Debugger is active!
2019-01-03 08:58:34,197 WARNING _internal.py:88  * Debugger is active!
2019-01-03 09:00:13,083 WARNING _internal.py:88  * Debugger is active!
2019-01-03 09:27:37,729 WARNING _internal.py:88  * Debugger is active!
2019-01-03 10:33:25,081 WARNING _internal.py:88  * Debugger is active!
2019-01-03 10:43:00,645 WARNING _internal.py:88  * Debugger is active!
2019-01-03 10:45:33,493 WARNING _internal.py:88  * Debugger is active!
2019-01-03 10:48:59,888 WARNING _internal.py:88  * Debugger is active!
2019-01-03 10:54:08,299 WARNING _internal.py:88  * Debugger is active!
2019-01-03 10:59:05,474 WARNING _internal.py:88  * Debugger is active!
2019-01-03 11:27:07,772 INFO houses.py:25 hit redis area_info
2019-01-03 13:54:23,820 WARNING _internal.py:88  * Debugger is active!
2019-01-03 13:55:10,100 WARNING _internal.py:88  * Debugger is active!
2019-01-03 14:32:32,997 INFO houses.py:25 hit redis area_info
2019-01-03 14:33:48,745 INFO houses.py:246 hit house index info redis
2019-01-03 14:33:48,748 INFO houses.py:25 hit redis area_info
2019-01-03 14:33:57,889 INFO houses.py:246 hit house index info redis
2019-01-03 14:33:57,892 INFO houses.py:25 hit redis area_info
2019-01-03 14:34:03,025 INFO houses.py:246 hit house index info redis
2019-01-03 14:34:03,028 INFO houses.py:25 hit redis area_info
2019-01-03 14:34:58,454 INFO houses.py:246 hit house index info redis
2019-01-03 14:34:58,646 INFO houses.py:25 hit redis area_info
2019-01-03 14:35:05,021 INFO houses.py:246 hit house index info redis
2019-01-03 14:35:05,245 INFO houses.py:25 hit redis area_info
2019-01-03 14:35:49,140 INFO houses.py:246 hit house index info redis
2019-01-03 14:35:49,312 INFO houses.py:25 hit redis area_info
2019-01-03 14:38:59,959 WARNING _internal.py:88  * Debugger is active!
2019-01-03 14:39:49,192 INFO houses.py:25 hit redis area_info
2019-01-03 14:40:15,720 INFO houses.py:246 hit house index info redis
2019-01-03 14:40:15,941 INFO houses.py:25 hit redis area_info
2019-01-03 14:40:19,367 INFO houses.py:246 hit house index info redis
2019-01-03 14:40:19,369 INFO houses.py:25 hit redis area_info
2019-01-03 14:46:58,580 WARNING _internal.py:88  * Debugger is active!
2019-01-03 14:47:05,494 INFO houses.py:247 hit house index info redis
2019-01-03 14:47:05,692 INFO houses.py:25 hit redis area_info
2019-01-03 14:47:09,242 INFO houses.py:247 hit house index info redis
2019-01-03 14:47:09,245 INFO houses.py:25 hit redis area_info
2019-01-03 14:49:50,477 WARNING _internal.py:88  * Debugger is active!
2019-01-03 14:49:53,312 INFO houses.py:248 hit house index info redis
2019-01-03 14:49:53,491 INFO houses.py:25 hit redis area_info
2019-01-03 14:53:47,058 INFO houses.py:248 hit house index info redis
2019-01-03 14:53:47,163 INFO houses.py:25 hit redis area_info
2019-01-03 14:59:20,295 WARNING _internal.py:88  * Debugger is active!
2019-01-03 14:59:22,576 INFO houses.py:248 hit house index info redis
2019-01-03 14:59:22,846 INFO houses.py:25 hit redis area_info
2019-01-03 14:59:29,841 INFO houses.py:248 hit house index info redis
2019-01-03 14:59:29,863 INFO houses.py:25 hit redis area_info
2019-01-03 15:04:33,333 INFO houses.py:248 hit house index info redis
2019-01-03 15:04:33,599 INFO houses.py:25 hit redis area_info
2019-01-03 15:04:39,560 INFO houses.py:248 hit house index info redis
2019-01-03 15:04:39,570 INFO houses.py:25 hit redis area_info
2019-01-03 15:05:48,004 INFO houses.py:248 hit house index info redis
2019-01-03 15:05:48,310 INFO houses.py:25 hit redis area_info
2019-01-03 15:09:39,090 INFO houses.py:248 hit house index info redis
2019-01-03 15:09:39,285 INFO houses.py:25 hit redis area_info
2019-01-03 15:10:44,281 INFO houses.py:248 hit house index info redis
2019-01-03 15:10:44,381 INFO houses.py:25 hit redis area_info
2019-01-03 15:12:39,375 INFO houses.py:248 hit house index info redis
2019-01-03 15:12:39,385 INFO houses.py:25 hit redis area_info
2019-01-03 15:13:28,498 INFO houses.py:248 hit house index info redis
2019-01-03 15:13:28,512 INFO houses.py:25 hit redis area_info
2019-01-03 15:15:20,821 WARNING _internal.py:88  * Debugger is active!
2019-01-03 15:15:38,897 WARNING _internal.py:88  * Debugger is active!
2019-01-03 15:16:03,936 INFO houses.py:248 hit house index info redis
2019-01-03 15:16:04,130 INFO houses.py:25 hit redis area_info
2019-01-03 15:16:17,894 INFO houses.py:248 hit house index info redis
2019-01-03 15:16:17,896 INFO houses.py:25 hit redis area_info
2019-01-03 15:16:22,876 INFO houses.py:248 hit house index info redis
2019-01-03 15:16:23,119 INFO houses.py:25 hit redis area_info
2019-01-03 15:16:53,400 WARNING _internal.py:88  * Debugger is active!
2019-01-03 15:16:56,327 INFO houses.py:248 hit house index info redis
2019-01-03 15:16:56,529 INFO houses.py:25 hit redis area_info
2019-01-03 15:17:22,446 WARNING _internal.py:88  * Debugger is active!
2019-01-03 15:18:01,979 WARNING _internal.py:88  * Debugger is active!
2019-01-03 15:18:15,355 WARNING _internal.py:88  * Debugger is active!
2019-01-03 15:18:22,378 INFO houses.py:248 hit house index info redis
2019-01-03 15:18:22,387 INFO houses.py:25 hit redis area_info
2019-01-03 15:18:34,397 INFO houses.py:248 hit house index info redis
2019-01-03 15:18:34,399 INFO houses.py:25 hit redis area_info
2019-01-03 15:19:51,731 INFO houses.py:248 hit house index info redis
2019-01-03 15:19:51,893 INFO houses.py:25 hit redis area_info
2019-01-03 15:20:42,926 INFO houses.py:25 hit redis area_info
2019-01-03 15:22:02,354 INFO houses.py:248 hit house index info redis
2019-01-03 15:22:02,357 INFO houses.py:25 hit redis area_info
2019-01-03 15:36:58,107 INFO houses.py:302 hit house info redis
2019-01-03 15:37:06,853 INFO houses.py:302 hit house info redis
2019-01-03 15:37:49,320 WARNING _internal.py:88  * Debugger is active!
2019-01-03 15:37:50,611 INFO houses.py:304 hit house info redis
2019-01-03 15:38:35,779 INFO houses.py:304 hit house info redis
2019-01-03 15:38:55,889 INFO houses.py:304 hit house info redis
2019-01-03 15:53:28,972 INFO houses.py:304 hit house info redis
2019-01-03 15:53:52,588 INFO houses.py:304 hit house info redis
2019-01-03 15:55:09,710 INFO houses.py:304 hit house info redis
2019-01-03 15:55:14,561 INFO houses.py:304 hit house info redis
2019-01-03 15:58:50,106 INFO houses.py:304 hit house info redis
2019-01-03 16:00:32,773 INFO houses.py:304 hit house info redis
2019-01-03 16:01:03,708 INFO houses.py:248 hit house index info redis
2019-01-03 16:01:10,027 INFO houses.py:304 hit house info redis
2019-01-03 16:05:34,891 INFO houses.py:304 hit house info redis
2019-01-03 16:06:31,668 INFO houses.py:304 hit house info redis
2019-01-03 16:06:35,778 INFO houses.py:304 hit house info redis
2019-01-03 16:06:51,559 INFO houses.py:248 hit house index info redis
2019-01-03 16:06:51,563 INFO houses.py:25 hit redis area_info
2019-01-03 16:06:54,778 INFO houses.py:248 hit house index info redis
2019-01-03 16:06:54,782 INFO houses.py:25 hit redis area_info
2019-01-03 16:07:02,943 INFO houses.py:248 hit house index info redis
2019-01-03 16:07:02,946 INFO houses.py:25 hit redis area_info
2019-01-03 16:07:12,654 INFO houses.py:304 hit house info redis
2019-01-03 16:08:15,092 INFO houses.py:248 hit house index info redis
2019-01-03 16:08:15,095 INFO houses.py:25 hit redis area_info
2019-01-03 16:08:22,644 INFO houses.py:304 hit house info redis
2019-01-03 16:08:39,615 INFO houses.py:248 hit house index info redis
2019-01-03 16:08:39,619 INFO houses.py:25 hit redis area_info
2019-01-03 16:08:42,767 INFO houses.py:248 hit house index info redis
2019-01-03 16:08:42,768 INFO houses.py:25 hit redis area_info
2019-01-03 16:08:47,412 INFO houses.py:248 hit house index info redis
2019-01-03 16:08:47,416 INFO houses.py:25 hit redis area_info
2019-01-03 16:08:53,136 INFO houses.py:304 hit house info redis
2019-01-03 16:09:01,468 INFO houses.py:304 hit house info redis
2019-01-03 16:09:06,764 INFO houses.py:248 hit house index info redis
2019-01-03 16:09:06,768 INFO houses.py:25 hit redis area_info
